layouts/partials/json-schema/resolve-additional-types.html

```diff
@@ -3,6 +3,7 @@
   Finds and returns all nested objects, given a dict containing:
    * `schema`: a JSON schema object
    * `anchor_base`: a prefix to add to the HTML anchors generated for each object. If nil, no anchors are generated.
+   * `name`: optionally, a name to use for this object in error/warning messages.
```

Comment:
You may wish to mention that if `name` is not set, the object's `title` property (if present) is used instead.

Reply:
```suggestion
   * `name`: optionally, a name to use for this object in error/warning messages. If left unset,
      the object's `title` property is used (if present).
```